core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -436,12 +436,23 @@ oppia.directive('conversationSkin', [
             }
 
             if (ExplorationPlayerStateService.isStateTerminal(stateName)) {
-              ExplorationRecommendationsService.getRecommendedSummaryDicts(
-                ExplorationPlayerStateService.getAuthorRecommendedExpIds(
-                  stateName),
-                function(summaries) {
-                  $scope.recommendedExplorationSummaries = summaries;
-                });
+              var lastParameter = UrlService.getUrlParams();
+              if (lastParameter.parent) {
+                var parent = [];
+                parent.push(lastParameter.parent);
+                ExplorationRecommendationsService.getRecommendedSummaryDicts(
```

Comment:
How are you ensuring that clicking on this summary dict will include the correct parent parameter, in the case where we've gone down two levels and are now returning to the first level (where there's still one parent left)?

Reply:
Yes, I am doing that now, I thought of adding another attribute to existing summary tile to make sure links are right when clicked or should I create another tile similar to summary tile with this functionality?

Reply:
Done